### ISSUE TITLE
worked 12 hours ago but does not work no

### ISSUE THREAD

**Issue by nicks-codes** [zoony1337/TikTok-Shares-Botter#37]
no shares go through anymore

**Comment by RootHuayra** [comment]
Yeah TikTok was onto this very quickly.

**Comment by Nyumat** [comment]
^^ What @RootHuayra said is correct, unless you can find a way to make a new session (account)  on every iteration it's basically patched.  Y'all abused it HELLA so TikTok caught on and made it so that you actually need Oauth accounts now. I'm working on a workaround right now though, so don't lose hope yet. Idk what happened to the Repo creator .

**Comment by RootHuayra** [comment]
Also even if you sign into a valid account and use a mobile browser user agent it still won't count as a share. I'm not quite sure what they are using to detect if it's an actual mobile device or not but it's working well.

**Comment by kenqkio** [comment]
> 
> 
>  Y'all abused it HELLA so TikTok caught on 

haha i saw a video with 100m+ shares

**Comment by RootHuayra** [comment]
> > Y'all abused it HELLA so TikTok caught on
> 
> haha i saw a video with 100m+ shares

Yeah I saw that to, tbh probably one of the main reasons they solved it so quickly.

**Comment by Nyumat** [comment]
I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.

**Comment by nicks-codes** [comment]
ya thanks tell me if anyone get it to work


discord
nicks#8151
or just reply

**Comment by nicks-codes** [comment]
> > Y'all abused it HELLA so TikTok caught on
> 
> haha i saw a video with 100m+ shares

same
its at 160 now

**Comment by RootHuayra** [comment]
> I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.

I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.

**Comment by nicks-codes** [comment]
> I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.

every time I run the code I get 1 share

**Comment by RootHuayra** [comment]
> > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> 
> every time I run the code I get 1 share

Just tried it, doesn't do that for me.

**Comment by kenqkio** [comment]
is there a system of token on tiktok, to just spam copy link with lot of accounts ?

**Comment by Nyumat** [comment]
i'm testing that right now @kenqkio

**Comment by TheLDB** [comment]
> > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> 
> I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.

Bridging off this comment would it be possible to run the script in an emulator environment?

**Comment by RootHuayra** [comment]
> > > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> > 
> > 
> > I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.
> 
> Bridging off this comment would it be possible to run the script in an emulator environment?

I mean you can try but I don't think it will work.

**Comment by TheLDB** [comment]
> > > > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> > > 
> > > 
> > > I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.
> > 
> > 
> > Bridging off this comment would it be possible to run the script in an emulator environment?
> 
> I mean you can try but I don't think it will work.

Wait since theres a copy link button on the web browser anyways what if we created a chrome extension to copy the link, would that still work? and if it does, wouldn't it be non patchable unless they removed the copy link button from chrome

**Comment by RootHuayra** [comment]
> > > > > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> > > > 
> > > > 
> > > > I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.
> > > 
> > > 
> > > Bridging off this comment would it be possible to run the script in an emulator environment?
> > 
> > 
> > I mean you can try but I don't think it will work.
> 
> Wait since theres a copy link button on the web browser anyways what if we created a chrome extension to copy the link, would that still work? and if it does, wouldn't it be non patchable unless they removed the copy link button from chrome

No, the Copy Link method only works if it is on an actual mobile device. Even changing the user agent to IOS / Android wont work.

**Comment by TheLDB** [comment]
> > > > > > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> > > > > 
> > > > > 
> > > > > I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.
> > > > 
> > > > 
> > > > Bridging off this comment would it be possible to run the script in an emulator environment?
> > > 
> > > 
> > > I mean you can try but I don't think it will work.
> > 
> > 
> > Wait since theres a copy link button on the web browser anyways what if we created a chrome extension to copy the link, would that still work? and if it does, wouldn't it be non patchable unless they removed the copy link button from chrome
> 
> No, the Copy Link method only works if it is on an actual mobile device. Even changing the user agent to IOS / Android wont work.

Oh odd, alright I'll take a look at it and try to fix it if no one else has by then

**Comment by RootHuayra** [comment]
> > > > > > > I wonder if the TikTok developers made it so ALL iterative sharing is patched. Like before u were still able to spam Share>CopyLink 100s of times (without code). If that still works, I tbh, could just make a selenium automation tool that keeps doing that on a bait account. Hmm. I'll look into it more.
> > > > > > 
> > > > > > 
> > > > > > I was thinking of selenium before but then I did some testing and changing the User Agent to a mobile device or using any sort of emulator doesn't work, but just using my phone to mass share does. I think you need actual mobile hardware in order to perform this kind of thing now.
> > > > > 
> > > > > 
> > > > > Bridging off this comment would it be possible to run the script in an emulator environment?
> > > > 
> > > > 
> > > > I mean you can try but I don't think it will work.
> > > 
> > > 
> > > Wait since theres a copy link button on the web browser anyways what if we created a chrome extension to copy the link, would that still work? and if it does, wouldn't it be non patchable unless they removed the copy link button from chrome
> > 
> > 
> > No, the Copy Link method only works if it is on an actual mobile device. Even changing the user agent to IOS / Android wont work.
> 
> Oh odd, alright I'll take a look at it and try to fix it if no one else has by then

I'm about to see what HTTP requests my phone makes to see if it is any different to the emulators.

**Comment by taikosrevenge** [comment]
One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.

**Comment by RootHuayra** [comment]
> One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.

I mean that's improvement from 0.

**Comment by taikosrevenge** [comment]
> > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> 
> I mean that's improvement from 0.

I guess so, but for people with like 8GB RAM you can barely run up to 4 instances and it slows down your pc so you can't really do anything while they're on.

**Comment by Nyumat** [comment]
> One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.

I'm doing something similar, but I didn't get results that slow. Because im using a throwaway account to do the shares on my main account, I noticed that the views iterate much faster than the shares, (yeah ik its more steps so thats why) but 5k in 2h might just be due to your computer processing speed. Using selenium and chrome web driver im getting 10k in 5 mins so far.

**Comment by nicks-codes** [comment]
> One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.

is there a way we can make a jailbreak tweak that does it for ios

**Comment by Nyumat** [comment]
I'm trying multiple strategies to figure out the fastest iteration method

**Comment by Nyumat** [comment]
> > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> 
> is there a way we can make a jailbreak tweak that does it for ios

Possibly, but I don't really want to have to learn another language lmao

**Comment by RootHuayra** [comment]
> > > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> > 
> > 
> > is there a way we can make a jailbreak tweak that does it for ios
> 
> Possibly, but I don't really want to have to learn another language lmao

Lmao was about to say the same thing.

**Comment by nicks-codes** [comment]
@Nyumat if u get selenium to work would u be able to show us how to make it work

**Comment by RootHuayra** [comment]
> > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> 
> I'm doing something similar, but I didn't get results that slow. Because im using a throwaway account to do the shares on my main account, I noticed that the views iterate much faster than the shares, (yeah ik its more steps so thats why) but 5k in 2h might just be due to your computer processing speed. Using selenium and chrome web driver im getting 10k in 5 mins so far.

What User Agent u using?

**Comment by nicks-codes** [comment]
> > > > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> > > 
> > > 
> > > is there a way we can make a jailbreak tweak that does it for ios
> > 
> > 
> > Possibly, but I don't really want to have to learn another language lmao
> 
> Lmao was about to say the same thing.

i don't really know much at all about coding

**Comment by nicks-codes** [comment]
> > > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> > 
> > 
> > I'm doing something similar, but I didn't get results that slow. Because im using a throwaway account to do the shares on my main account, I noticed that the views iterate much faster than the shares, (yeah ik its more steps so thats why) but 5k in 2h might just be due to your computer processing speed. Using selenium and chrome web driver im getting 10k in 5 mins so far.
> 
> What User Agent u using?

if yall figure it out can yo show us how if works, unless its just a code u download and run

**Comment by crazymoose77756** [comment]
> > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> 
> is there a way we can make a jailbreak tweak that does it for ios

well there are tweaks like autotouch but its kinda pointless on your phone, you can't do much else while its going

**Comment by kenqkio** [comment]
but whats about the share limit per account ?

**Comment by TheLDB** [comment]
> but whats about the share limit per account ?

Are we sure theres a share limit?

**Comment by RootHuayra** [comment]
> but whats about the share limit per account ?

I don't think they have one.

**Comment by kenqkio** [comment]
i heard that there is a limit of 100 but i didnt checked

**Comment by TheLDB** [comment]
> i heard that there is a limit of 100 but i didnt checked

The one guy above in the thread said he got 2k with 4 emulators so I dont think theres a limit

**Comment by ycuglu** [comment]
> i'm testing that right now @kenqkio

ay bro tell me when u finish i'm trying to execute a plan on thatveganteacher so we can save bella

**Comment by kenqkio** [comment]
> thatveganteacher
hahaha

**Comment by RootHuayra** [comment]
> > One way is to run an autoclicker on multiple Bluestacks instances but the shares are slow. I got like 5k in 2h.
> 
> I'm doing something similar, but I didn't get results that slow. Because im using a throwaway account to do the shares on my main account, I noticed that the views iterate much faster than the shares, (yeah ik its more steps so thats why) but 5k in 2h might just be due to your computer processing speed. Using selenium and chrome web driver im getting 10k in 5 mins so far.

My selenium browser doesn't even let me load tiktok posts, it says they're unavailable and even after changing the user agent.

**Comment by crazymoose77756** [comment]
ok well progress is being made!

**Comment by nicks-codes** [comment]
how does selenium work

**Comment by RootHuayra** [comment]
> how does selenium work

It is a python and javascript library that allows you to control your browser.

**Comment by Nyumat** [comment]
Guys I messed around with the .py file a little bit and got a View Bot working. Send me your TikTok usernames so I can try it

**Comment by Nyumat** [comment]
This is groundbreaking :p

**Comment by kenqkio** [comment]
> 
> 
> Guys I messed around with the .py file a little bit and got a View Bot working. Send me your TikTok usernames so I can try it

kenanzebi

**Comment by Nyumat** [comment]
Which video do you want it on

**Comment by RootHuayra** [comment]
just do his latest

**Comment by kenqkio** [comment]
yeah its just for testing

**Comment by nicks-codes** [comment]
slappaayy do the latest

**Comment by nicks-codes** [comment]
> Guys I messed around with the .py file a little bit and got a View Bot working. Send me your TikTok usernames so I can try it

if it works can u send us it

**Comment by Nyumat** [comment]
> > Guys I messed around with the .py file a little bit and got a View Bot working. Send me your TikTok usernames so I can try it
> 
> kenanzebi thats a alt account with not many views

Check the HunterXHunter Video. It should be working

**Comment by RootHuayra** [comment]
it worked

**Comment by crazymoose77756** [comment]
> it worked

lets go

**Comment by Nyumat** [comment]
Ayeeee

**Comment by kenqkio** [comment]
yeah instant 600+ views

**Comment by taikosrevenge** [comment]
> Ayeeee

gj

**Comment by nicks-codes** [comment]
> Ayeeee

can u give us the code

**Comment by nicks-codes** [comment]
good jod dude

**Comment by kenqkio** [comment]
what you did ? @Nyumat

**Comment by Nyumat** [comment]
I'm still working on scaling it so it's faster and can take more  queries (views per sec)

**Comment by nicks-codes** [comment]
> I'm still working on scaling it so it's faster and can take more queries (views per sec)

ok sick

**Comment by Nyumat** [comment]
I might try and make a website so it's easier for us to do it. So like at the click of a button idk

**Comment by Nyumat** [comment]
TikTok detects the bots if its over 1k a second so I might put a limiter on it. Idk, its efficient but I just want to optimize it so it cant be patched like the share bot was.

**Comment by kenqkio** [comment]
you're being to nice dude

**Comment by nicks-codes** [comment]
bet

**Comment by PhilippeDr** [comment]
Can we get the code for this?

**Comment by nicks-codes** [comment]
does it work the same way like u type the views and then the url

**Comment by Nyumat** [comment]
> you're being to nice dude

Trying to get hired by Tech Companies so I rather not make it for profit

**Comment by Nyumat** [comment]
> does it work the same way like u type the views and then the url

No it takes your username and automatically adds the views to whichever video you click. I'm coding up a client so we can just do it at the click of a button give me some time. :p

**Comment by H43X** [comment]
We can put it down in a discord bot, thats limited to 1 server

**Comment by Nyumat** [comment]
> > does it work the same way like u type the views and then the url
> 
> No it takes your username and automatically adds the views to whichever video you click. I'm coding up a client so we can just do it at the click of a button give me some time. :p

Well atleast this is what I intend. Rn it takes the url.

**Comment by crazymoose77756** [comment]
> > > does it work the same way like u type the views and then the url
> > 
> > 
> > No it takes your username and automatically adds the views to whichever video you click. I'm coding up a client so we can just do it at the click of a button give me some time. :p
> 
> Well atleast this is what I intend. Rn it takes the url.

sounds great!

**Comment by Nyumat** [comment]
> We can put it down in a discord bot, thats limited to 1 server

Can someone make a discord server or like snapchat group chat lmao

**Comment by kenqkio** [comment]
i can create discord if you want

**Comment by PhilippeDr** [comment]
Can you just keep it as URL that would seem to work good enough and its currently not patched, Ide rather do it before it gets patched

**Comment by nicks-codes** [comment]
ya i got it

**Comment by ycuglu** [comment]
> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

I can see if i can do it

**Comment by RootHuayra** [comment]
> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

lmao yeah this is the biggest github convo i've ever had

**Comment by H43X** [comment]
> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

I'm good at making bots so I can add it into my bot

**Comment by bobdabiulder** [comment]
> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

Add me Bobbo Banano#6983

**Comment by kenqkio** [comment]
go for it

**Comment by nicks-codes** [comment]
https://discord.gg/UEAnTW

**Comment by RootHuayra** [comment]
my discord account got stolen yesterday but ill join on my alt

**Comment by kenqkio** [comment]
> 
> 
> > > We can put it down in a discord bot, thats limited to 1 server
> > 
> > 
> > Can someone make a discord server or like snapchat group chat lmao
> 
> lmao yeah this is the biggest github convo i've ever had

hahah same

**Comment by bobdabiulder** [comment]
> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao
We're all in here:
https://discord.gg/UEAnTW

**Comment by ycuglu** [comment]
Numyat who is you?

> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

**Comment by nicks-codes** [comment]
ya idk

**Comment by nicks-codes** [comment]
lol

**Comment by ycuglu** [comment]
> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

bruh make it public doesnt it defeat the whole purpose of github if you dont have it up?

**Comment by nicks-codes** [comment]
@

> > We can put it down in a discord bot, thats limited to 1 server
> 
> Can someone make a discord server or like snapchat group chat lmao

join

**Comment by picomachaca** [comment]
> @
> 
> 
> 
> > > We can put it down in a discord bot, thats limited to 1 server
> 
> > 
> 
> > Can someone make a discord server or like snapchat group chat lmao
> 
> 
> 
> join
> 
> 

Can you make the link public?

**Comment by rmukondiwa** [comment]
im trying to find a way to make a new sharebot should i make a discord?

**Comment by picomachaca** [comment]
> im trying to find a way to make a new sharebot should i make a discord?
> 
> 

Yeah, you Could, you could also work together

**Comment by lkspq** [comment]
Suggestion. If you find a way to fix the problem, you shoudn't pull the code publicly. Pretty sure a dev from bytedance is following this repo. It won't last a day.

**Comment by kyv0n** [comment]
> https://discord.gg/UEAnTW

drop a new link for me lol

**Comment by Nyumat** [comment]
> im trying to find a way to make a new sharebot should i make a discord?

We have an org going. PM for link.

**Comment by Nyumat** [comment]
> Suggestion. If you find a way to fix the problem, you shoudn't commit the code publicly. Pretty sure a dev from bytedance is following this repo. It won't last a day.

Yeah, literally every method we've come up with has been patched by ByteDance the minute we do it lmao.

**Comment by MadCrazyMoosey** [comment]
> ^^ What @RootHuayra said is correct, unless you can find a way to make a new session (account) on every iteration it's basically patched. Y'all abused it HELLA so TikTok caught on and made it so that you actually need Oauth accounts now. I'm working on a workaround right now though, so don't lose hope yet. Idk what happened to the Repo creator .

Nice! Will it go public if you do the work around?

**Comment by MadCrazyMoosey** [comment]
> > im trying to find a way to make a new sharebot should i make a discord?
> 
> We have an org going. PM for link.

What is the org link?

**Comment by kyv0n** [comment]
> > im trying to find a way to make a new sharebot should i make a discord?
> 
> We have an org going. PM for link.

how do you even pm on here?

**Comment by picomachaca** [comment]
> > > im trying to find a way to make a new sharebot should i make a discord?
> 
> > 
> 
> > We have an org going. PM for link.
> 
> 
> 
> how do you even pm on here?

Ik lmao, I want also the link

**Comment by 21iot** [comment]
> We have an org going. PM for link.
Ayo add me, I know a bit about this stuff and I think I can help solve this. I actually fixed the UA error like right when it happened after sniffing requests on my Nox emulator, and I created a discord bot  to automatically do what this program does. I think I've narrowed it down to that you just need a sessionid in your cookies for the share to go through

**Comment by dbohlman** [comment]
> https://discord.gg/UEAnTW

let me get another link

**Comment by Nyumat** [comment]
I'll invite you guys to the org. Check your inboxes.

**Comment by JackTYM** [comment]
> I'll invite you guys to the org. Check your inboxes.

could i get one too

**Comment by Mad-houdini8** [comment]
> I'll invite you guys to the org. Check your inboxes.

Yo, can I also get one also?

**Comment by dgorbunov** [comment]
@Nyumat could I get an invite as well? I think we should all try out a couple different methods and see which ones are most efficient in share botting.

**Comment by dbohlman** [comment]
> I'll invite you guys to the org. Check your inboxes.

yeah me as well

**Comment by picomachaca** [comment]
> I'll invite you guys to the org. Check your inboxes.

How do I check my inbox?

**Comment by dbohlman** [comment]
> > I'll invite you guys to the org. Check your inboxes.
> 
> How do I check my inbox?

click the bell

**Comment by Mattkimmio** [comment]
> I'll invite you guys to the org. Check your inboxes.

Me too PLEASE

**Comment by slashccjr** [comment]
> I'll invite you guys to the org. Check your inboxes.

hey can i get the link too?

**Comment by lkspq** [comment]
pls message me the link or invitation. Discord Joe100#2693

**Comment by valbtw** [comment]
> I'll invite you guys to the org. Check your inboxes.

Could you also add me to it? My Discord is:   valqzo#6345
Thank you!

**Comment by lkspq** [comment]
> > Suggestion. If you find a way to fix the problem, you shoudn't commit the code publicly. Pretty sure a dev from bytedance is following this repo. It won't last a day.
> 
> Yeah, literally every method we've come up with has been patched by ByteDance the minute we do it lmao.

If I found this repo, so the devs from bytedance. I studied the code, it was a simple, but engineous solution. Also I'm willing to help for educational purposes. It's cool learning about this social networks APIs.

**Comment by kenqkio** [comment]
https://discord.gg/TtrsDNc hey here is  a perm link to the discord we are working on it ;)

**Comment by henmasgat** [comment]
> I'll invite you guys to the org. Check your inboxes.

can u mail me it? discord: Henmasgat#3180

**Comment by nicks-codes** [comment]
https://discord.gg/KZwA3x



That’s the link

**Comment by nicks-codes** [comment]
> > https://discord.gg/UEAnTW
> 
> drop a new link for me lol

https://discord.gg/QuvsE6Q

**Comment by nicks-codes** [comment]
> > I'll invite you guys to the org. Check your inboxes.
> 
> Could you also add me to it? My Discord is: valqzo#6345
> Thank you!

https://discord.gg/QuvsE6Q

**Comment by nicks-codes** [comment]
> @Nyumat could I get an invite as well? I think we should all try out a couple different methods and see which ones are most efficient in share botting.

https://discord.gg/QuvsE6Q

**Comment by nicks-codes** [comment]
> > I'll invite you guys to the org. Check your inboxes.
> 
> hey can i get the link too?

https://discord.gg/QuvsE6Q

**Comment by nicks-codes** [comment]
> > We have an org going. PM for link.
> 
> Ayo add me, I know a bit about this stuff and I think I can help solve this. I actually fixed the UA error like right when it happened after sniffing requests on my Nox emulator, and I created a discord bot to automatically do what this program does. I think I've narrowed it down to that you just need a sessionid in your cookies for the share to go through

https://discord.gg/QuvsE6Q

**Comment by nicks-codes** [comment]
https://discord.gg/QuvsE6Q

that the discord we have 50 people working join if u can help

**Comment by PragmaticCoder** [comment]
hey can you email me the link as well. My email is pheonixhax007@gmail.com

**Comment by nicks-codes** [comment]
https://discord.gg/Q9WB6pu

**Comment by liamthedux124** [comment]
> > > Suggestion. If you find a way to fix the problem, you shoudn't commit the code publicly. Pretty sure a dev from bytedance is following this repo. It won't last a day.
> > 
> > 
> > Yeah, literally every method we've come up with has been patched by ByteDance the minute we do it lmao.
> 
> If I found this repo, so the devs from bytedance. I studied the code, it was a simple, but engineous solution. Also I'm willing to help for educational purposes. It's cool learning about this social networks APIs.

Pls do me discord : Liam The Dux #2070

**Comment by zoony1337** [comment]
For the people asking, it has been patched.
